Source/Rows/PickerRow.swift

```diff
@@ -67,6 +67,10 @@ open class PickerCell<T> : Cell<T>, CellType, UIPickerViewDataSource, UIPickerVi
         textLabel?.text = nil
         detailTextLabel?.text = nil
         picker.reloadAllComponents()
+        selectValue()
```

Comment:
I would prefer to create a _PickerCell class and  override `update()` method for each case.